### PR TITLE
Style/#229 landing page lower grey section css

### PR DESCRIPTION
**Pull request by daniel-cy-lu**
Lower grey section (including the three cubes) and footer responsive css completed.

Mock-up: https://www.figma.com/file/fXh58MfxEDh1ssgxQyPb5f/Overture_Refresh?type=design&node-id=1167-9957&mode=design&t=8yIZjhY1lHQIveER-0

### CODE REVIEW

**Comment by daniel-cy-lu** [comment]
@ciaranschutte @mistryrn Hi guys, usually Sam helps me review these responsive css, but she has a new assignment and @joneubank suggested that I tag you. For this ticket, I completed the lower grey section in the home/landing page of overture. I provided a link to the mock up in the description. Thank you for your help as always! :D

File: src/components/Footer/index.js [hunk 1]
```diff
@@ -19,86 +19,73 @@ import { NETLIFY_LINK, NETLIFY_IMAGE_LINK, OICR_LINK } from 'constants/external-
...
-          <Link className="column-container__link mt1 mb1" to={topItem.link}>
-            {topItem.name}
-          </Link>
-          <Link className="column-container__link mt1 mb1" to={bottomItem.link}>
-            {bottomItem.name}
-          </Link>
-        </div>
+    <div className="links-container">
+      {links.map((link) => (
+        <Link className="links-container__link" to={link.url}>
```
Comment (ciaranschutte) [inline src/components/Footer/index.js:86]: you need to put a `key` prop in these links so React can reconcile its renders.
fixes this error:

Screenshot 2023-08-21 at 11 15 29 AM

File: src/components/Footer/styles.scss [hunk 2]
```diff
@@ -7,68 +7,128 @@
...
+      gap: 40px;
+      padding-left: 2.5em;
+    }
 
     &__oicr-logo {
       width: 138px;
       height: auto;
-      margin-left: -10px;
+      @include breakpoint(tablet-up) {
+        // width: auto;
```
Comment (ciaranschutte) [inline src/components/Footer/styles.scss:27]: this is not doing anything because it's commented out. can remove whole `@include` segment

**Review by ciaranschutte — CHANGES_REQUESTED** [review]
two small changes.
I tested it out locally, looking really good @daniel-cy-lu - responsiveness is great!

File: src/components/Footer/index.js (hunk 1 above)
Comment (daniel-cy-lu) [reply src/components/Footer/index.js:86]: I will take a look, thanks Ciaran! didn't see this before.

File: src/components/Footer/styles.scss (hunk 2 above)
Comment (daniel-cy-lu) [reply src/components/Footer/styles.scss]: removed used css

File: src/components/Footer/index.js (hunk 1 above)
Comment (ciaranschutte) [reply src/components/Footer/index.js:86]: I don't see an error in the screenshot? is it the yellow underline? 
These would likely be TS errors. Check to see if an error appears in the console of the browser please.

best practice to at least do:
```
{links.map((link, index) => <Link key={index} ...other code/>
```
or some always unique part of the data

more info can be found here: https://react.dev/learn/rendering-lists

Comment (ciaranschutte) [reply src/components/Footer/index.js:86]: just double checked console - should be ok to add a key to resolve error - let me know if there's still an issue

Comment (daniel-cy-lu) [reply src/components/Footer/index.js:86]: Thanks Ciaran for showing the best practice. I add index to the map method in <FooterLink> and all the error in your initial screenshot went away! 

I thought I had to go and find each <Link> and add the key! :)

**Comment by daniel-cy-lu** [comment]
@joneubank Thanks Jon

**Pull request merged**